pkg/cmd/server/bootstrappolicy/constants.go

```diff
@@ -70,6 +70,8 @@ const (
 	BuildStrategySourceRoleName          = "system:build-strategy-source"
 	BuildStrategyJenkinsPipelineRoleName = "system:build-strategy-jenkinspipeline"
 
+	RestrictedEndpointsAdmissionRoleName = "system:endpoints-admission"
```

Comment:
instead of this, we want a role, associated with a service account, that actually includes all the permissions the endpoints controller needs. I assume that is get/list/watch access to services and pods, full access to endpoints, and now create access to our virtual `endpoints/restricted` resource (or whatever we call it)